CriticalMapsSnapshotTests/XCTestCase+SnapshotTests.swift

```diff
@@ -31,7 +32,7 @@ extension XCTestCase {
                            as: .image(precision: precision, size: size),
                            named: theme.displayName,
```

Comment:
because the theme display name is now localized, we get snapshot references with `*.Hell.png` and `*.Dunkel.png`. I'm thinking about using the raw value here instead of the display name. What do you think?
Otherwise we have to update snapshot references each time the localization for the theme display name changes.
```suggestion
                           named: theme.rawValue,
```

Reply:
Oh. Well you don't want that. The rawvalue makes sense. The other one should only be used for screen appearance

Reply:
Merged this from the app. Maybe snapshots need an update too